C7/PCXToGodot.cs

```diff
@@ -73,30 +110,24 @@ public class PCXToGodot : Godot.Object
 	//Combines two PCXs, one used for the alpha, to produce a final output image.
 	//Some files, such as Art/interface/menuButtons.pcx and Art/interface/menuButtonsAlpha.pcx, use this method.
 	public static ImageTexture getImageFromPCXWithAlphaBlend(Pcx imagePcx, Pcx alphaPcx, int leftStart, int topStart, int croppedWidth, int croppedHeight, int alphaRowOffset = 0) {
-		Image OutImage = new Image();
-		OutImage.Create(croppedWidth, croppedHeight, false, Image.Format.Rgba8);
-		OutImage.Lock();
-		for (int y = topStart; y < topStart + croppedHeight; y++)
-		{
-			for (int x = leftStart; x < leftStart + croppedWidth; x++)
-			{
-				int currentPixel = y * imagePcx.Width + x;
-				byte red = imagePcx.Palette[imagePcx.ColorIndexAt(x, y), 0];
-				byte green = imagePcx.Palette[imagePcx.ColorIndexAt(x, y), 1];
-				byte blue = imagePcx.Palette[imagePcx.ColorIndexAt(x, y), 2];
-				//Assumption based on menuButtonsAlpha.pcx: The palette in the alpha PCX always has the same red, green, and blue values (i.e. is grayscale).
-				//Examining it with breakpoints in my Java code, it appears it starts at 255, 255, 255, and goes down one at a time.  But this code
-				//doesn't assume that, it only assumes the grayscale aspect.  In theory, this should work for any transparency, 0 to 255.
-				byte alpha = alphaPcx.Palette[alphaPcx.ColorIndexAt(x, y - alphaRowOffset), 0];
-				OutImage.SetPixel(x - leftStart, y - topStart, Color.Color8(red, green, blue, alpha));
-			}
-		}
-		OutImage.Unlock();
+		loadPalette(imagePcx.Palette);
+		loadAlphaPalette(alphaPcx.Palette);
 
-		ImageTexture Txtr = new ImageTexture();
-		Txtr.CreateFromImage(OutImage, 0);
-		return Txtr;
+		int yEnd = topStart + croppedHeight;
+		int xEnd = leftStart + croppedWidth;
```

Comment:
Minor, but in cases like this where yEnd/xEnd are only used in one place and are simple, I don't think there's any gain to separating them out.  If we're reading line 122, it's quicker to read, "y goes from topStart to topStart + croppedHeight... okay, so it's a range of size croppedHeight starting at topStart", than to read, "y goes from topStart to yEnd... now what's yEnd?  Oh right, it's topStart + croppedHeight... so the difference is croppedHeight, so it's a range from topStart of length croppedHeight".  It just winds up being a different line to read to read what could be on one line easily.

Obviously there comes a point when separating things out _does_ make sense (see the junk byte comments in Pcx.cs), but if it's just an offset from the start, I think it's easier to have it in-line.  As for any performance difference, an optimizing compiler will see that topStart/croppedHeight never change, and make the upper bound of the for loop be a constant, so the performance will be identical.